src/matsim_io/writers.py

```diff
@@ -166,3 +161,23 @@ def add_leg_with_route(
         self.indent -= 1
 
         self._write_line("</leg>")
+
+
+def _compute_capacity(
+    speed_limit: float,
+    vehicle_length: float = 5,
+    min_gap: float = 2.5,
+    tau: float = 1,
+) -> float:
+    """
+    Compute the capacity of a link based on its speed limit.
+    :param speed_limit: Maximum allowed speed of the link in meters per second.
+    :param vehicle_length: Physical length of a vehicle in meters.
+    :param min_gap: Minimum gap between vehicles in a standing queue in meters.
+    :param tau: Desired minimum time headway in seconds.
```

Comment:
I still don't get what tau means?

Reply:
It's described in the SUMO reference in the same docstring. I just copied their method.